Frontend/person-detection/components/ui-accordion-helm/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
```

Comment:
Classes inside /components folder from [Spartan](https://www.spartan.ng) library.

Reply:
There are a lot of new changes, but basically they stem from Spartan components and package.json.